### ISSUE TITLE
showFront method

### ISSUE THREAD

**Issue by akeane1950** [mgechev/jqcarousel#3]
Hello!  I just came upon your jQcarousel v1.1.5 and started playing with it for a project I have in mind.

I would like to use the showFront method but I keep on getting function not defined.

I have used jQuery for a while but am not great at it.  I know I am doing something silly but I have tried some things and the best I know is:  $('#gallery_87to93').showFront(1).  Thus to show the fourth image in my gallery.  Seems it defaults to the last image and I really want to show the first one first.

Any help would be appreciated.

Thanks.

**Comment by akeane1950** [comment]
Okay... I got it to work by specifying:
    $('#gallery_87to93').jqcarousel('showFront',0);
Got the idea from StackOverflow:
    http://stackoverflow.com/questions/7246769/add-custom-method-to-ui-dialog-plugin

If there is another way to 'call' it please let me know.

Thanks...

**Comment by mgechev** [comment]
Yes, this should be fine.

Thanks for giving the project a try :-)

**Comment by akeane1950** [comment]
Thank you for your response.  I have been learning a lot from your plugin.  Besides accessing a method I figured out how to access variables since I wanted to get the current image information. 

I have one carousel working with show and hide and now I have to work on having more than one and to generalize my routines.  This has been a great learning experience because your code is very straight forward well... at least the part I looked at so far.

I do have at least two more questions about working with this plugin.  I am not sure where to ask them.  Any suggestions?  I may run across more in the future.

Thanks again for a very enjoyable plugin.

TTFN... Andrea
